lib/collection/src/operations/types.rs

```diff
@@ -596,8 +596,8 @@ pub struct DiscoverRequest {
     /// Look for vectors closest to this
     pub target: Option<RecommendExample>,
```

Comment:
Maybe something for the future: if we would want to extend this to the recommendation-style positive and negative lists, how would API look like?

Reply:
Untagged enum of RecommendExample + Vec<RecommendExample> ? or would you rather have new type altogether?